### PR TITLE
Add new examples to documentation

### PR DESCRIPTION
**Pull request by anzoman**
We try to stick to the plan in #121 and therefore gradually update our
documentation. After we have resurrected opera-examples repository,
populating it with a lot of new TOSCA templates, we can now describe
some of the most interesting ones in the documentation.

### CODE REVIEW

**Comment by anzoman** [comment]
The connected PR from `xopera-examples` repo: https://github.com/xlab-si/xopera-examples/pull/7.

**Comment by anzoman** [comment]
Anotherconnected PR from xopera-examples repo: xlab-si/xopera-examples#9.

**Comment by anzoman** [comment]
Both connected PRs (xlab-si/xopera-examples#7 and xlab-si/xopera-examples#9) were reviewed and merged so this PR is fully prepared for review.

**Review by cankarm — APPROVED** [review]

File: docs/examples.rst [hunk 1]
```diff
@@ -34,21 +35,168 @@ in playbooks that implement life cycle operations.
...
 
-
 ============
 Public cloud
 ============
 
-TBD
-
+xOpera and Ansible open up a great possibility to deploy solutions on different public cloud providers such as
+Amazon Web Services (AWS), Microsoft Azure (Azure), Google Cloud Platform (GCP), OpenFaaS.
```
[the remark this answers is not in the document]
Comment (anzoman) [reply docs/examples.rst]: OpenFaaS has been moved to the private cloud examples.

**Comment by anzoman** [comment]
@cankarm because you approved the PR with additional comment I had to resolve it first and then force push the commits again which means that you will have to approve again (if everything is ok).

**Pull request merged**